src/swarm-tui/Swarm/TUI/View/Shared.hs

```diff
@@ -0,0 +1,15 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+-- |
+-- SPDX-License-Identifier: BSD-3-Clause
+--
+-- UI view components shared across dialogs
+module Swarm.TUI.View.Shared where
+
+import Brick
+import Brick.Widgets.Center (hCenter)
+import Swarm.TUI.Model.Name
+import Swarm.TUI.View.Attribute.Attr (italicAttr)
+
+tabControlFooter :: Widget Name
+tabControlFooter = hCenter $ withAttr italicAttr $ txt "NOTE: [Tab] toggles focus between panes"
```

Comment:
If you make this polymorphic in the `Name` it could go to `Swarm.TUI.View.Util`, I think.

But maybe we will need a common set of shared view parts anyway.